packages/splits-kit/tsconfig.json

```diff
@@ -2,11 +2,9 @@
   "extends": "../../tsconfig.json",
   "baseUrl": "./src",
   "compilerOptions": {
+    "module": "ESNext",
```

Comment:
I was running into an error importing this library, which was a result of using wagmi (an es module) in a commonjs `require("wagmi")` statement. 

Making our library compile to es module fixed this issue.

I believe this is the way to go for javascript run in the browser.

For more info:
https://www.tsmean.com/articles/learn-typescript/typescript-module-compiler-option/